.appveyor.yml

```diff
@@ -104,7 +104,7 @@ install:
     # install composer dependencies
     - cd C:\projects\dbal
     - appveyor-retry composer self-update
-    - appveyor-retry composer install --no-progress --prefer-dist
+    - appveyor-retry composer update --no-progress --prefer-dist
```

Comment:
One of the reasons why we commit the lock file is to have a stable set of development dependencies. With this approach, as soon as a non-major update of a tool introduces a check that the DBAL will not pass, we'll have a failing build.

Reply:
Yes, for doctrine/orm we pinned the development tools to exact versions for this reason. I will improve this here in the PR as well.